tests/TestWebApp/appsettings.config

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<appSettings>
+    <add key="zipkinBaseUri" value="https://zipkin-sandbox.imedidata.net" />
```

Comment:
I think we had better remove this project from the repository or change some configuration values.